packages/to-dts/lib/types/function.js

```diff
@@ -2,33 +2,18 @@
 const dom = require('dts-dom');
 const params = require('./params');
 
-module.exports = function fn(def, tsParent, g) {
+module.exports = function fn(def, tsParent, g, isEntry) {
   const par = params(def.params || [], def.this, g);
   const ret = def.returns ? g.getType(def.returns) : dom.type.void;
+  if (isEntry) {
+    return dom.create.function(def.name, par, ret);
+  }
   if (tsParent && ['class', 'interface', 'object'].includes(tsParent.kind)) {
     return dom.create.method(def.name, par, ret);
   }
   if (tsParent && ['union', 'array', 'parameter', 'alias'].includes(tsParent.kind)) {
     return dom.create.functionType(par, ret);
   }
```

Comment:
consider placing the isEntry test after the parent checks.
(it isEntry is never true for thoes cases it will be ok as it is)

Reply:
I have moved the `if (isEntry) {}` downwards. Have not yet seen any specification where it occurs but better to play it safe.